### DISCUSSION TITLE
Welcome to git-bash-for-mac Discussions!

### DISCUSSION THREAD

**Discussion by Ondkloss** [Ondkloss/git-bash-for-mac#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by stardustman** [comment]
Thanks the feature for macOS zsh

**Reply by Ondkloss** [reply]
> Thanks the feature for macOS zsh

Thank you for your kind feedback 😊
